### PR TITLE
Citadel's folder's end

### PR DESCRIPTION
**Pull request by Poojawa**
:cl: Poojawa
del: Removed old things in .dms we weren't using anymore
tweak: subtle messages are now italic'd, so there's better context on whispering
tweak: broke apart the cit_gun.dm file, they're decently spaced out now
tweak: dogborg_sleeper is now standalone from dogborg_equipment because too lazy to debug why it was breaking backpacks.
tweak: does that thing I've been meaning to do with Xenobio. additional tools have been provided.
/:cl:

modernizing and shuffling. wew

### CODE REVIEW

**Comment by Poojawa** [comment]
Also this will likely accumulate merge conflicts as it sits.

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -129,7 +129,7 @@
 	display_name = "Basic Bluespace Theory"
 	description = "Basic studies into the mysterious alternate dimension known as bluespace."
 	prereq_ids = list("base")
-	design_ids = list("beacon", "xenobioconsole")
+	design_ids = list("beacon")
```
Comment (deathride58) [inline code/modules/research/techweb/all_nodes.dm]: //CIT CHANGE

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -148,7 +148,7 @@
 	display_name = "Applied Bluespace Research"
 	description = "Using bluespace to make things faster and better."
 	prereq_ids = list("bluespace_basic", "engineering")
-	design_ids = list("bs_rped","minerbag_holding", "telesci_gps", "bluespacebeaker", "bluespacesyringe", "bluespacebodybag", "phasic_scanning", "roastingstick")
+	design_ids = list("bs_rped","minerbag_holding", "telesci_gps", "bluespacebeaker", "bluespacesyringe", "bluespacebodybag", "phasic_scanning", "roastingstick", "xenobioconsole")
```
Comment (deathride58) [inline code/modules/research/techweb/all_nodes.dm]: // CIT CHANGE

**Review by deathride58 — CHANGES_REQUESTED** [review]
Seems like everythings's in working order

**Review by deathride58 — APPROVED** [review]

**Pull request merged**